Source/Randomizer.cpp

```diff
@@ -147,15 +147,14 @@ void Randomizer::Randomize() {
         // be solveable in the down position after using Sigma's randomizer. We
         // do not want to swap any such panel with Tutorial Back Left, because
         // that would make it impossible to exit Tutorial.
-        const std::set<int> bannedTutorialBackLeftPanels = {
+        std::vector<int> upDownPanelsSetThreeDoubleMode = copyWithoutElements(upDownPanelsSetThree, {
             0x00070, // Symmetry Island Fading Lines 5
             0x01E5A, // Mill Entry Door Left
             0x00072, // Symmetry Island Fading 3
             0x00076, // Symmetry Island Fading 7
             0x3C125, // Mill Control Room Extra Panel
             0x09E85, // Tunnels Into Town Big Triangles
```

Comment:
I think we called this panel Tunnels Town Shortcut? Not really picky, just wondering if the name is different elsewhere.

Reply:
In splits I usually see it as UTM Town Shortcut; this name is just the name I copied out of Panels.h.

Actually, looking into Panels.h, I see all three names being used in various places. I could change them all to be consistent if you want.

Reply:
I would love that, thank you. I don't know if we're using UTM or Tunnels in this project, but "Town Shortcut" is definitely the right name.